src/context/modal-context.tsx

```diff
@@ -31,11 +32,13 @@ const ModalProvider: FC<ModalProviderProps> = ({ children }) => {
   const [modal, setModal] = useState<React.ReactElement | null>(null)
   const [paperProps, setPaperProps] = useState<PaperProps>({})
   const [timer, setTimer] = useState<NodeJS.Timeout | null>(null)
+  const [isOpen, setIsOpen] = useState<boolean>(true)
 
   const closeModal = useCallback(() => {
     setModal(null)
     setPaperProps({})
     setTimer(null)
+    setIsOpen(true)
```

Comment:
I guess on close it should be false, and default value in state should be false

Reply:
not relevant